sbin/spark-daemon.sh

```diff
@@ -195,6 +195,23 @@ case $option in
     fi
     ;;
 
+  (status)
+
+    if [ -f $pid ]; then
+      TARGET_PID=`cat $pid`
+      if kill -0 $TARGET_PID > /dev/null 2>&1; then
```

Comment:
See how the process is detected in the rest of this file. The problem with this is that theoretically you would report the process is running even if, somehow, it had died and the PID was still around but another process had taken on the same ID. It's possible to be more defensive. Heck, if you know enough `bash`, make a local function for it.